create.js

```diff
@@ -1,9 +1,31 @@
-import { pushSection } from ./src/lib/debug.js;
+import { pushSection } from './styles/lib/db.js'
```

Comment:
Why do you put the db.js in a `styles` folder? db.js has nothing to do with styles or am I missing something? ;-)